package.json

```diff
@@ -45,4 +45,4 @@
   "dependencies": {
     "ts-mobx-basic-stores": "^0.6.3"
   }
-}
\ No newline at end of file
+}
```

Comment:
Again, it's not me, it's NPM 🙈

Reply:
If you want to install packages locally and don't affect the `package-lock.json` and `package.json` use [npm ci](https://docs.npmjs.com/cli/v6/commands/npm-ci) command. Which install dependencies from lock file only.

`npm install` command install and try to update dependencies accordingly to [semver expressions](https://jubianchi.github.io/semver-check/#/).

Reply:
@terrasoff Should I exclude changes in `package-lock.json` and `package.json`? Most of the developers would run `npm install` after cloning the repo, it shouldn't affect neither `package.json` or `package-lock.json`. For example if you clone Mobx repo and run `yarn install` package.json remains unchanged.